common/djangoapps/track/views/tests/test_views.py

```diff
@@ -1,22 +1,27 @@
 # pylint: disable=missing-docstring,maybe-no-member
 
-from track import views
-from track.middleware import TrackMiddleware
 from mock import patch, sentinel
-from freezegun import freeze_time
```

Comment:
This is used elsewhere - it's a handy library that makes datetime.now() return a consistent value that you specify without having to mock it all over the place.